### ISSUE TITLE
Latest release broke elm-doc-test when using it as a devDependency (Cannot find module './elm.js')

### ISSUE THREAD

**Issue by lovasoa** [stoeffel/elm-doc-test#17]
elm-doc-test doesn't work anymore when used as a devDependency.

### Broken package
Example from https://github.com/lovasoa/elm-format-number

### package.json

```json
{
  "name": "elm-format-number",
  "version": "0.0.0",
  "description": "Format numbers as pretty strings",
  "main": "index.js",
  "directories": {
    "test": "tests"
  },
  "scripts": {
    "test": "elm-doc-test && elm-test init && elm-test tests/Doc/Main.elm"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/lovasoa/elm-format-number.git"
  },
  "keywords": [
    "format",
    "elm",
    "number",
    "int",
    "float"
  ],
  "author": "",
  "license": "BSD-3-Clause",
  "bugs": {
    "url": "https://github.com/lovasoa/elm-format-number/issues"
  },
  "homepage": "https://github.com/lovasoa/elm-format-number#readme",
  "devDependencies": {
    "elm": "^0.18.0",
    "elm-doc-test": "^2.1.1",
    "elm-test": "^0.18.2"
  }
}
```

#### with 2.1 (works)
```
olojkine@ordophir:~/Developpement/elm-format-number$ npm install --save-dev elm-doc-test@2.1
elm-format-number@0.0.0 /home/olojkine/Developpement/elm-format-number
└── elm-doc-test@2.1.1 

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.0.17: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
olojkine@ordophir:~/Developpement/elm-format-number$ npm test

> elm-format-number@0.0.0 test /home/olojkine/Developpement/elm-format-number
> elm-doc-test && elm-test init && elm-test tests/Doc/Main.elm

Running in generate mode..
/home/olojkine/Developpement/elm-format-number/src already exists
/home/olojkine/Developpement/elm-format-number/tests already exists
/home/olojkine/Developpement/elm-format-number/tests/elm-package.json already exists
/home/olojkine/Developpement/elm-format-number/tests/Main.elm already exists
/home/olojkine/Developpement/elm-format-number/tests/Tests.elm already exists
/home/olojkine/Developpement/elm-format-number/tests/.gitignore already exists
Success! Compiled 4 modules.                                        
Successfully generated /tmp/elm_test_11711-26795-1kfn9rl.43mfs9k9.js

elm-test
--------

Running 26 tests. To reproduce these results, run: elm-test --seed 1253949742


TEST RUN PASSED

Duration: 26 ms
Passed:   26
Failed:   0
```

#### With 2.2 (broken)
```
olojkine@ordophir:~/Developpement/elm-format-number$ npm install --save-dev elm-doc-test@2.2
elm-format-number@0.0.0 /home/olojkine/Developpement/elm-format-number
└── elm-doc-test@2.2.0 

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.0.17: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
olojkine@ordophir:~/Developpement/elm-format-number$ npm test

> elm-format-number@0.0.0 test /home/olojkine/Developpement/elm-format-number
> elm-doc-test && elm-test init && elm-test tests/Doc/Main.elm

module.js:442
    throw err;
    ^

Error: Cannot find module './elm.js'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/olojkine/Developpement/elm-format-number/node_modules/elm-doc-test/bin/modes.js:5:11)
    at Module._compile (module.js:541:32)
    at Object.Module._extensions..js (module.js:550:10)
    at Module.load (module.js:458:32)
    at tryModuleLoad (module.js:417:12)
    at Function.Module._load (module.js:409:3)
npm ERR! Test failed.  See above for more details.
```

**Comment by stoeffel** [comment]
@lovasoa I republished 2.2.1. Could you try again?

**Comment by lovasoa** [comment]
It didn't change anything. You can test yourself by cloning the repo mentioned above

**Comment by lovasoa** [comment]
[`bin/modes.js` requires `./elm.js`](https://github.com/stoeffel/elm-doc-test/blob/master/bin/modes.js#L5) but this file [is not present in the repo](https://github.com/stoeffel/elm-doc-test/tree/master/bin)

**Comment by stoeffel** [comment]
okay, it's fixed now in v2.2.3. I had to add https://github.com/stoeffel/elm-doc-test/blob/master/package.json#L12 to include elm.js in the published package. Thanks for the hint.
